modules/apps/29-fee/ibc_module.go

```diff
@@ -175,8 +175,19 @@ func (im IBCModule) OnRecvPacket(
 	ctx sdk.Context,
 	packet channeltypes.Packet,
 	relayer sdk.AccAddress,
-) ibcexported.Acknowledgement {
-	// TODO: Implement fee specific logic if fee is enabled for the given channel
+) exported.Acknowledgement {
+	if im.keeper.IsFeeEnabled(ctx, packet.SourcePort, packet.SourceChannel) {
```

Comment:
I think this should be destination port and channel since this is on the receiving side. Could we ensure the tests using different channel id's so that such an bug would be caught?

Reply:
good catch

Reply:
This seems to not have been fixed